### ISSUE TITLE
content of /info

### ISSUE THREAD

**Issue by ghost** [ONLYOFFICE/DocumentServer#612]
05.07 Peak Mouth = 16
12.07 Peak Mouth = 13 
12.07 peak week   =  8 

how does this work?

and on which date the stats get reset?

Prio 0

**Comment by ShockwaveNN** [comment]
@hahaxd35 Sorry, currently we have no documentation about `/info` page. We'll add some info about that

**Comment by hahaxd35** [comment]
Okay its Look Like
Week = last 7 days 
mouth = last 30 days 

but we can use this statistic for license analytic?

**Comment by ShockwaveNN** [comment]
@hahaxd35 I'm not sure what do you mean by license analytic, but I think you can use this data :)

**Comment by hahaxd35** [comment]
license analytic means that this kind of statistic is equal to the license of  "Simultaneous connections "

**Comment by ShockwaveNN** [comment]
@hahaxd35 yep, if I understand your right. This page is developed especially to analyze best license to buy

**Comment by hahaxd35** [comment]
Okay thanks! that it ticket can be closed? or wait till documentation?

**Comment by ShockwaveNN** [comment]
@hahaxd35 We're working on documentation now and I'll close issue as soon as we release it

**Comment by igwyd** [comment]
We'll keep you posted as soon as there's an official guide available.

**Comment by trofim24** [comment]
Starting from version 9.1, the ONLYOFFICE Admin Panel provides a secure web interface for managing server configuration and monitoring. You can monitor these statistics more easily through the web interface.
See more - https://helpcenter.onlyoffice.com/docs/installation/docs-admin-panel.aspx
